acceptance-tests/src/main/kotlin/org/ostelco/at/loadtest/OcsLoadTest.kt

```diff
@@ -17,35 +34,177 @@ import org.ostelco.prime.customer.model.BundleList
 import java.time.Instant
 import java.util.*
 import java.util.concurrent.CountDownLatch
+import java.util.concurrent.Executor
+import java.util.concurrent.Executors
 import kotlin.streams.toList
 import kotlin.test.assertEquals
 import kotlin.test.fail
 
-class OcsLoadTest {
+class OcsGrpcLoadTest {
```

Comment:
Will rename it back to OcsLoadTest since it now has load test for both, gRPC and PubSub.